_source/_includes/promo_banner.html

```diff
@@ -12,7 +12,7 @@
       <svg id="Layer_2" xmlns="http://www.w3.org/2000/svg" width="26px" height="32px" viewBox="0 0 26 32"><defs><style>.cls-1{fill:#fffefa;}</style></defs><g id="Layer_1-2"><g><path class="cls-1" d="M13.48,3.28c-1.9,.69-3.89,1.04-5.91,1.04h-.82c-1.84,0-3.56,.73-4.84,2.05C.62,7.69-.05,9.43,0,11.28c.05,1.66,.75,3.17,1.84,4.33l.12,9.21H6.43l2.08-6.96c1.7,.09,3.37,.42,4.98,1l9.02,3.28V0L13.48,3.28ZM2.98,7.42c1-1.03,2.34-1.59,3.77-1.59h.82c.82,0,1.63-.07,2.43-.17v10.85c-.8-.1-1.61-.17-2.43-.17h-.57c-2.95,0-5.42-2.28-5.5-5.09-.04-1.43,.48-2.79,1.48-3.82Zm2.33,15.91h-1.86l-.08-6.51c1.05,.63,2.28,.99,3.59,1l-1.64,5.5Zm15.69-3.32l-7-2.55c-.82-.3-1.65-.52-2.5-.7V5.39c.84-.18,1.68-.4,2.5-.7l7-2.55V20Z"/><g><polygon class="cls-1" points="24 9.2 28.14 6.71 27.36 5.43 24 7.45 24 9.2"/><polygon class="cls-1" points="24 14.7 27.36 16.71 28.14 15.43 24 12.95 24 14.7"/><rect class="cls-1" x="24" y="10.32" width="5.75" height="1.5"/></g></g></g></svg>
-      <span class="promo-body">Coming May 22, 2025! Okta is introducing a new free org for developers and integrators. Click this banner to learn more.</span>
+      <span class="promo-body">Now Available! Okta is introducing a new free org for developers and integrators. Click this banner to learn more.</span>
```

Comment:
Should the text match dev.okta and devforum?

> Integrator Free Plan available! Existing Okta Developer Edition orgs will be deactivated starting July 18, 2025. Sign up for the Integrator Free Plan to continue building and integrating. Learn more on the [Okta Developer blog](https://developer.okta.com/blog/2025/05/13/okta-developer-edition-changes).

Reply:
Saw the new dev forum banner text and came here immediately to make it match =)